tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MacroSuite.scala

```diff
@@ -24,34 +24,101 @@ class MacroSuite extends BaseDottySuite {
   }
 
   test("macro-quote-expr") {
+    runTestAssert[Stat]("'{ 'ax }")(
+      Term.QuotedMacroExpr(Term.Block(List(Term.QuotedMacroExpr(tname("ax")))))
+    )
+    runTestAssert[Stat]("'{ 'a + 'b }", assertLayout = Some("'{ 'a + ('b) }"))(
```

Comment:
```suggestion
    runTestAssert[Stat]("'{ 'a + 'b }", assertLayout = Some("'{ 'a + 'b }"))(
```
?
why would we add () to the second one?

Reply:
we don't in this case, fixed